### PR TITLE
Add DarwiinRemoteOSC 0.6

### PR DESCRIPTION
**Pull request by leafac**
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/darwiin-remote-osc.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'darwiin-remote-osc' do
+  version '0.6'
+  sha256 'f98c6c982182ae899af0e1f729120242544e5b3231002802aaef8d565068fcb9'
```
Comment (miccal) [inline Casks/darwiin-remote-osc.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`.

File: Casks/darwiin-remote-osc.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'darwiin-remote-osc' do
```
Comment (vitorgalvao) [inline Casks/darwiin-remote-osc.rb]: Token reference was not followed.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/darwiinremoteosc.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+cask 'darwiinremoteosc' do
+  version '0.6'
```
Comment (vitorgalvao) [inline Casks/darwiinremoteosc.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]

**Comment by leafac** [comment]
Thank you all for the fixes and for merging my contribution! 👍

**Pull request merged**